controller/rest/volume.go

```diff
@@ -36,6 +39,37 @@ func (s *Server) GetVolume(rw http.ResponseWriter, req *http.Request) error {
 	return nil
 }
 
+func (s *Server) GetReplicaInfo() []types.ReplicaInfo {
+	var (
+		info []types.ReplicaInfo
+		wg   sync.WaitGroup
+	)
+	infoLock := &sync.Mutex{}
+	replicas := s.c.ListReplicas()
```

Comment:
Need to take a lock and copy the list in a different variable.
Range might have unexpected behavior if a replica is removed from the list by some other function.

Reply:
Good catch.